Modules/_zstd/_zstdmodule.h

```diff
@@ -1,132 +1,30 @@
-#pragma once
 /*
 Low level interface to Meta's zstd library for use in the compression.zstd
 Python module.
 */
 
 /* Declarations shared between different parts of the _zstd module*/
 
+#ifndef ZSTD_MODULE_H
```

Comment:
I'd suggest having a `ZSTD_ZSTDMODULE_H` so to mimick the file path. This allows for a possible future `Modules/_zstdmodule.h` file

Reply:
We could potentially also rename to `_module.{ch}` similar to sqlite? I think the guard define isn't public API here so we can change it if we want, but I'm not fully sure.

Reply:
We have `_io/_iomodule.[ch]`. I think it's better to keep the full name because it helps when looking for the file.